ondewo/vtsi/dataclasses/server_configurations_dataclasses.py

```diff
@@ -40,10 +40,6 @@ class AudioConfiguration:
     s2t_port: int = 443
     s2t_type: str = "ONDEWO"
 
-    # demux settings
-    demux_host: str = "0.0.0.0"
-    demux_port: int = 8011
-
     language_code: Optional[str] = None
     t2s_language: Optional[str] = None
     s2t_language: Optional[str] = None
```

Comment:
i don;t see the point to have language of stt and tts

Reply:
because the vtsi server creates the csi server and the pipeline

how else would you set it?

oh, i think 'language' here actually means language model.

Reply:
oh, i remember now, it should be s2t and t2s pipeline, not the language. Language is outdated naming